tap-gui/plugins/runtime-resource-visibility.md

```diff
@@ -66,15 +65,82 @@ applicable for the resource type.
 To view the list of your running resources:
 
 1. Select the Component from the Catalog index page.
-1. Select the Workloads tab.
+2. Select the Runtime Resources tab.
 
-    ![Workload index table](./images/workload-visibility-workloads.png)
+![Runtime resources index table collapsed](images/runtime-resources-index.png)
 
-## Knative service details page
+### Seeing details for a specific resource
+
+The index table will show Deployments, Pods, ReplicaSets and Services that matches with the label indicated in the component's definition; you will see a master-detail structure between all the different objects using the owners and dependents relationships stablished between them; those resources without an owner will be listed in the table as independent elements.
+
+> For additional information about owners and dependants please visit [the official documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/)
+
+Following, you can see an example of an expanded view for the index table.
+
+![Runtime resources index table collapsed](images/runtime-resources-expanded.png)
+
+## Detail pages
+
+Currently, we provide pages for many resources to allows users to see the most relevant characteristics of each one, including direct links to other resources.
+These are the common sections included in all pages:
+
+### Overview section
+
+It is the first card in all details pages, most of the information included in it came from the `metadata` attribute in each object. Some attributes displayed here are:
+
+1. **.YAML** button: Show the current object's definition in yaml; you can copy the full content using the icon in the top-right corner.
```

Comment:
When you click on the .YAML button, a side panel opens showing the current object's definition in yaml. You can copy the full content of the .YAML file by using the icon in the top-right corner of the side panel.